cluster/gce/gci/master-helper.sh

```diff
@@ -76,6 +76,12 @@ function replicate-master-instance() {
 function create-master-instance-internal() {
   local gcloud="gcloud"
   local retries=5
+  local sleep_sec=10
+  if [[ "${MASTER_SIZE}" =~ -64$ ]]; then
```

Comment:
Is this the biggest MASTER_SIZE?
maybe?
```
local suffix=$( echo "${MASTER_SIZE}" | sed 's/.*-\([0-9]\+\)/\1' )
if [[ "${suffix}" -ge 64 ]]; then
```

Reply:
Good point, this will be more robust.
Done.